sdks/python/apache_beam/examples/inference/pytorch_image_classification.py

```diff
@@ -114,10 +115,13 @@ def run(argv=None, model_class=None, model_params=None, save_main_session=True):
     model_class = MobileNetV2
     model_params = {'num_classes': 1000}
 
-  model_loader = PytorchModelLoader(
-      state_dict_path=known_args.model_state_dict_path,
-      model_class=model_class,
-      model_params=model_params)
+  # the input to RunInference transform is keyed. Wrap
```

Comment:
This sounds like the input is always keyed. Isn't it user's choice? If so, would this be appropriate instead:

```
In this example we pass keyed inputs to RunInference transform.
Therefore, we use KeyedModelHandler wrapper over PytorchModelHandler.
```

Reply:
Done